lib/babl/builder/template_base.rb

```diff
@@ -9,8 +14,16 @@ def initialize(builder = ChainBuilder.new(&:itself))
                 @builder = builder
             end
 
-            def compile(**options)
-                Rendering::CompiledTemplate.new(precompile, **options)
+            def compile(preloader: Rendering::NoopPreloader, pretty: true)
```

Comment:
does this mean that for bannerman we should have pretty false?

imo false would be a better default